gpiozero/boards.py

```diff
@@ -2237,3 +2237,98 @@ def off(self):
         """
         self.buzzer.value = None
         super(JamHat, self).off()
+
+class Pibrella(CompositeOutputDevice):
+    """
+    Extends :class:`CompositeOutputDevice` for the `Cyntech/Pimoroni Pibrella`_ board.
+
+    There are 3 LEDs, one button, a tonal buzzer, 4 outputs (with LEDs) and 4 inputs. The pins are fixed.
+    Usage::
+
+        from gpiozero import Pibrella
+
+        pb = Pibrella()
+
+        pb.lights.on()
+        pb.button.wait_for_press()
+        pb.outputs.on()
+        pb.buzzer.play('A4')
+        pb.inputs.a.wait_for_press()
+        pb.off()
+
+    :param bool pwm:
+        If :data:`True`, construct :class: PWMLED instances to represent each
+        LED on the board. If :data:`False` (the default), construct regular
+        :class:`LED` instances.
+
+    :type pin_factory: Factory or None
+    :param pin_factory:
+        See :doc:`api_pins` for more information (this is an advanced feature
+        which most users can ignore).
+
+    .. _Pibrella: https://www.pibrella.com/
+
+    .. attribute:: lights, outputs
+
+        :class:`LEDBoard` instance representing the three LEDs  
+
+        .. attribute:: red, yellow, green
+
+            :class:`LED` or :class:`PWMLED` instances representing the red,
+            yellow, and green LEDs .
+
+    .. attribute:: outputs
+    
+        :class:`LEDBoard` instance representing the 4 Out headers on the Pibrella.  
```

Comment:
I'm not sure LEDBoard is appropriate here... it's a bit uglier code-wise, but I think a CompositeOutputDevice of DigitalOutputDevices would be better. (and also shows that theses are 'generic' outputs and not 'LED' outputs?)

Reply:
I have no strong preference,  LEDBoard fits the need (and there are LEDs attached to the outputs) and follows the pattern of the original API.  I'll look at the suggestion you've made, not been that deep into GPIOZero yet :-)